docs/components/base/list.tsx

```diff
@@ -1,10 +1,9 @@
 import { css } from '@emotion/core';
+import { theme } from '@expo/styleguide';
```

Comment:
Maybe we can just export the `UL` and `LI` from `@expo/styleguide` for this component (and maybe others)?

Reply:
I intend to do that for many components, just haven't added them to the styleguide lib yet.